app/views/tasks/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<strong>Details:</strong>
+<p>Task Name: <%= @task[:name] %></p>
```

Comment:
You should use the dot methods here rather than this kind of hash syntax; `@task.name` instead